src/status_im/ui/screens/chat/message/datemark.cljs

```diff
@@ -8,5 +8,7 @@
    {:on-press (fn [_]
                 (react/dismiss-keyboard!))}
    [react/view style/datemark-mobile
+
     [react/text {:style (style/datemark-text)}
-     (string/capitalize value)]]])
```

Comment:
Hi, shouldn't we use quo2/text component which already has these text-styles defined
![image](https://user-images.githubusercontent.com/17097240/178483874-46f71a20-1ad3-472f-b64f-66f08693b354.png)